modelator/cli/__init__.py

```diff
@@ -413,6 +413,11 @@ def _load_model_with_arguments(
 
     model = None
 
+    if os.path.exists(model_path) == False:
+        raise RuntimeError(
+            f"Model file {model_path} does not exist"
+        )
+
```

Comment:
This code block should be under `if model_path:` (L422). Because this function takes `None` as `model_path`.